src/kvstore/raftex/RaftPart.cpp

```diff
@@ -263,7 +263,15 @@ void RaftPart::start(std::vector<HostAddr>&& peers, bool asLearner) {
                         << " copies. The quorum is " << quorum_ + 1
                         << ", as learner " << asLearner;
 
-    committedLogId_ = lastCommittedLogId();
+    auto logIdAndTerm = lastCommittedLogId();
+    committedLogId_ = logIdAndTerm.first;
+    term_ = proposedTerm_ = logIdAndTerm.second;
+
+    if (lastLogId_ < committedLogId_) {
+        lastLogId_ = committedLogId_;
+        lastLogTerm_ = term_;
+        wal_->reset(lastLogId_ + 1);
```

Comment:
Maybe reset to committedLogId_ + 1?

Reply:
I had removed the param passed in in reset interface. 
The interface just clean up all wall files. So we don't need the logId here.